noodles-editor/src/noodles/operators.ts

```diff
@@ -4142,6 +4141,354 @@ export class GeoJsonTransformOp extends Operator<GeoJsonTransformOp> {
+    const fromPoint = turf.point([from.lng, from.lat])
+    const toPoint = turf.point([to.lng, to.lat])
+
+    const distance =
+      method === 'rhumb'
+        ? turf.rhumbDistance(fromPoint, toPoint, { units })
+        : turf.distance(fromPoint, toPoint, { units })
+
+    return { distance }
+  }
+}
+
+export class RadiansToDegreesOp extends Operator<RadiansToDegreesOp> {
+  static displayName = 'RadiansToDegrees'
+  static description = 'Convert radians to degrees'
+  createInputs() {
+    return {
+      radians: new NumberField(0, { step: 0.01 }),
+    }
+  }
+  createOutputs() {
+    return {
+      degrees: new NumberField(),
+    }
+  }
+  execute({ radians }: ExtractProps<typeof this.inputs>): ExtractProps<typeof this.outputs> {
+    const degrees = turf.radiansToDegrees(radians)
+    return { degrees }
+  }
+}
+
+export class DegreesToRadiansOp extends Operator<DegreesToRadiansOp> {
+  static displayName = 'DegreesToRadians'
+  static description = 'Convert degrees to radians'
+  createInputs() {
+    return {
+      degrees: new NumberField(0, { step: 1 }),
+    }
+  }
+  createOutputs() {
+    return {
+      radians: new NumberField(),
+    }
+  }
+  execute({ degrees }: ExtractProps<typeof this.inputs>): ExtractProps<typeof this.outputs> {
+    const radians = turf.degreesToRadians(degrees)
+    return { radians }
+  }
+}
+
+export class CentroidOp extends Operator<CentroidOp> {
+  static displayName = 'Centroid'
+  static description = 'Calculate the centroid of a GeoJSON feature'
+  asDownload = () => this.outputData
+  createInputs() {
+    return {
+      feature: new FeatureField(),
+      properties: new DataField({}),
+    }
+  }
+  createOutputs() {
+    return {
+      centroid: new FeatureField(),
+    }
+  }
+  execute({
+    feature,
+    properties,
+  }: ExtractProps<typeof this.inputs>): ExtractProps<typeof this.outputs> {
+    const centroid = turf.centroid(feature, { properties })
+    return { centroid }
+  }
+}
+
+export class AreaOp extends Operator<AreaOp> {
+  static displayName = 'Area'
+  static description = 'Calculate the area of a GeoJSON feature in square meters'
+  createInputs() {
+    return {
+      feature: new FeatureField(),
+    }
+  }
+  createOutputs() {
+    return {
+      area: new NumberField(),
+      areaKm2: new NumberField(),
+      areaMi2: new NumberField(),
+    }
+  }
+  execute({ feature }: ExtractProps<typeof this.inputs>): ExtractProps<typeof this.outputs> {
+    const area = turf.area(feature)
+    const areaKm2 = area / SQ_METERS_TO_SQ_KM
+    const areaMi2 = area / SQ_METERS_TO_SQ_MILES
+
+    return { area, areaKm2, areaMi2 }
+  }
+}
+
+export class BufferOp extends Operator<BufferOp> {
+  static displayName = 'Buffer'
+  static description = 'Create a buffer around a GeoJSON feature'
+  asDownload = () => this.outputData
+  createInputs() {
+    return {
+      feature: new FeatureField(),
+      radius: new NumberField(1, { min: 0, max: 10000, step: 0.1 }),
```

Comment:
This doesn't have a consistent hard max.. it'd vary by unit and even then it's not clear what it'd actually be